spec/services/marketing_site_spec.rb

```diff
@@ -3,26 +3,58 @@
 RSpec.describe MarketingSite do
   describe '.base_url' do
     it 'points to the base URL' do
-      expect(MarketingSite.base_url).to eq('https://www.login.gov')
+      expect(MarketingSite.base_url).to eq('https://www.login.gov/')
+    end
+
+    context 'when the user has set their locale to :es' do
+      it 'points to the base URL with the locale appended' do
+        I18n.locale = :es
+
+        expect(MarketingSite.base_url).to eq('https://www.login.gov/es/')
+      end
     end
   end
 
   describe '.privacy_url' do
     it 'points to the privacy page' do
       expect(MarketingSite.privacy_url).to eq('https://www.login.gov/policy')
     end
+
+    context 'when the user has set their locale to :es' do
+      it 'points to the privacy page with the locale appended' do
+        I18n.locale = :es
```

Comment:
I would move this into a `before` block (so it's part of the `context`) and the same for all of these:

```ruby
context 'when the user has their locale set to :es' do
  before { I18n.locale = :es }

  it 'points to the privacy page with the locale embedded' do
   expect # ...
  end
end
```